WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package org.wordpress.android.ui.jetpack.backup.details
+
+import android.view.ViewGroup
+import androidx.annotation.MainThread
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsViewModel.ListItemUiState
+import org.wordpress.android.ui.utils.UiHelpers
+
+class BackupDownloadDetailsAdapter(private val uiHelpers: UiHelpers) :
+        RecyclerView.Adapter<BackupDownloadDetailsViewHolder>() {
+    private val items = mutableListOf<ListItemUiState>()
+
+    override fun onCreateViewHolder(
```

Comment:
1) Minor (🔍 ): Consider preserving the order of the override functions, just like they appear on the parent class:
- `onCreateViewHolder(...)`
- `onBindViewHolder(...)`
- `getItemId(...)`
- `getItemCount()`
2) Minor (🔍 ) For all those functions consider converting to expression body and removing the explicit type specification (`: Long`). This will make them a one-liner (ish).